pygmt/base_plotting.py

```diff
@@ -135,6 +136,23 @@ def coast(self, **kwargs):
         shorelines : str
             ``'[level/]pen'``
             Draw shorelines [Default is no shorelines]. Append pen attributes.
+        dcw : str or list
+            ``code1,code2,…[+l|L][+gfill][+ppen][+z]``
```

Comment:
Please update the syntax following #631.

Reply:
Think I fixed it.